### PR TITLE
Feat: Special for serverless config

### PR DESCRIPTION
**Pull request by mzl-md**

### CODE REVIEW

File: src/special/serverless.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  return plugins;
+}
+
+export default async function parseServerless(filename) {
+  const basename = path.basename(filename);
+
+  if (basename === 'serverless.yml') {
+    const content = await getContent(filename);
+    const config = yaml.safeLoad(content);
+    // TODO find plugins defined in included files
```
Comment (rumpl) [inline src/special/serverless.js:27]: This todo sems like it's been done no?

**Comment by rumpl** [comment]
Hey @mzl-md thanks for the PR, just a minor comment but LGTM overall

File: src/special/serverless.js (hunk 1 above)
Comment (mzl-md) [reply src/special/serverless.js:27]: This detects only plugins from the main serverless.yml, but I think you could have plugins in included files like this:
```
plugins: ${file(./serverless.plugins.yml)}
```

Comment (rumpl) [reply src/special/serverless.js:27]: Oooh, ok great, could you add that comment in the TODO so that we know what is not implemented?

Comment (mzl-md) [reply src/special/serverless.js]: Done 👍

**Comment by rumpl** [comment]
@mzl-md sorry to bother you, could you just rebase with master and then we can merge it.

**Comment by mzl-md** [comment]
Now the checks failed with `Error: Cannot find module '../dist/cli'`, but that isn't something I caused, is it?

**Comment by rumpl** [comment]
> Now the checks failed with `Error: Cannot find module '../dist/cli'`, but that isn't something I caused, is it?

Re-running the checks, surely something flaky

**Pull request merged**